### ISSUE TITLE
Inconsistent Format of U.S. State Time Series Files

### ISSUE THREAD

**Issue by harryrowe01** [CSSEGISandData/COVID-19#1858]
The U.S. Deaths time series file has one more column, Population, than the U.S. Confirmed Case file.

**Comment by oltdaniel** [comment]
@harryrowe01 This dataset is the best one we can use for "official" figures. The data format changes each day due to countries reporting on different ways or different intervalls. It is hard to combine all sources into one, but @CSSEGISandData does an amazing job. We as developers are asked to write software that actually fits the changed data formats there in order to make our apps work. Else we need to write an custom interface as I did e.g. https://c.oltdaniel.at to suit my application needs. So please close this issue, as this is not the first on to complain about data format and @CSSEGISandData  does it's best to find a way to publish its data.

**Comment by loankimrobinson** [comment]
@oltdaniel I agree with @oltdaniel , @CSSEGISandData does the best they can, so many people complained about the format, and structure of data changed. After I saw my app didn't load, I knew there was something wrong with the data, I updated my app again. I think we need to maintain our apps as they need to update their data everyday. To me I am so happy, and thankful for using their data.
